### DISCUSSION TITLE
I think I made the living conditions in this world unlivable.. lol

### DISCUSSION THREAD

**Discussion by JustLearninglol** [niels747/2D-Weather-Sandbox#23] [General]
![image](https://user-images.githubusercontent.com/105594008/194211011-3d937aa1-6e20-44d5-8222-4c10a5e8ac08.jpg)

**Comment by niels747** [comment]
30 c dewpoint is unbearable, but it does happen  some in places on earth! Poor people! The highest dewpoint I ever experienced was 23 c
